spec/operations/mailers/user_added_to_project.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe Mailers::UserAddedToProject do
```

Comment:
this filename should change to ` spec/operations/mailers/user_added_to_project_spec.rb `